src/sas/qtgui/MainWindow/GuiManager.py

```diff
@@ -570,16 +573,15 @@ def processVersion(self, version_info):
                 msg += " Please try again later."
                 self.communicate.statusBarUpdateSignal.emit(msg)
 
-            elif version.__gt__(LocalConfig.__version__):
+            elif version.__gt__(sas.system.version.__version__):
```

Comment:
`sas.system.version.__version__` is imported as `SASVIEW_VERSION`. Any reason to not use the constant?

Reply:
Shouldn't really be imported like that. I saw it, but I have kept it in so as to avoid mission creep.